annotation/src/main/resources/application.properties.EXAMPLE

```diff
@@ -1,5 +1,6 @@
 vep.url=http://grch37.rest.ensembl.org/vep/human/hgvs/VARIANT?content-type=application/json&xref_refseq=1&ccds=1&canonical=1&domains=1&hgvs=1&numbers=1&protein=1
 hotspots.url=http://cancerhotspots.org/api/hotspots/single/
-vep.overrides_uris=uniprot=isoform_overrides_uniprot.txt,mskcc=isoform_overrides_at_mskcc.txt
+vep.overrides_uris=uniprot:isoform_overrides_uniprot.txt,mskcc:isoform_overrides_at_mskcc.txt
 
-spring.data.mongodb.uri=
+spring.data.mongodb.uri=mongodb://127.0.0.1:27017/annotator
```

Comment:
Since this is an example file, I think it is okay to have a working example value as long as it is not confidential. We can add a comment on top of each property to make things clearer.

@zheins What do you think?